sklearn/pipeline.py

```diff
@@ -657,6 +659,21 @@ def n_features_in_(self):
         # delegate to first step (which will call _check_is_fitted)
         return self.steps[0][1].n_features_in_
 
+    def __sk_is_fitted__(self):
```

Comment:
It is true that the pattern inside the function looks typical of a property.

Reply:
isn't that true for `_more_tags`?

Reply:
Yes it does as well :)